cmd/kube-controller-manager/app/bootstrap.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import "k8s.io/kubernetes/pkg/controller/bootstrap"
```

Comment:
I'm putting it there as it is a peer to the serviceaccount stuff.  In some ways these are very similar. if you have another place to put it, let me know.

As for making it external from KCM -- as this is part of cluster bootstrap I'm not sure what other process it would belong in. At the end of the day, we can't introduce a new daemon for every controller.  Well, we could, but it would be a bit crazy.  Binding a lot of controllers into the KCM is a necessary efficiency trade off.